### PR TITLE
Rollup Econ - Handle for Multiple OP Versions

### PR DESCRIPTION
**Pull request by MSilb7**
OP Mainnet batch submitters had used many combinations of to/from across versions, but the current 'Rollup Economics' spell only pulls the most recent combination, which leads to underreported L1 costs.

The OutputOracleProxy was also missed as the to address for state outputs (although this is super small).

This spell:
1. Adds `codebase` to `addresses_ethereum_l2_batch_submitters` for easier filtering (rather than maintaining a list of rollups).
2. Creates a view to generate all OP batch submitter address combinations (batchinbox and outputoracle combos), also handle for pulling the outpout oracle proxy for to addresses.
3. Adjusts the Rollup Econ spell to generate batchinbox and output oracle separately for Optimism chains

cc @Jam516 for eyes

### CODE REVIEW

**Comment by Jam516** [comment]
makes sense to me 👍

**Comment by jeff-dude** [comment]
marking as `wip` until CI tests are resolved

File: models/addresses/ethereum/addresses_ethereum_optimism_batchinbox_combinations.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{{ config(alias = 'optimism_batchinbox_combinations',
```
Comment (jeff-dude) [inline models/addresses/ethereum/addresses_ethereum_optimism_batchinbox_combinations.sql:1]: can you add `schema` in all config blocks?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/addresses/ethereum/addresses_ethereum_optimism_batchinbox_combinations.sql (hunk 1 above)
Comment (MSilb7) [reply models/addresses/ethereum/addresses_ethereum_optimism_batchinbox_combinations.sql:1]: on it

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**